src/demo/index.jsx

```diff
@@ -183,7 +183,7 @@ class Demo extends Component {
                 <button
                   className="btn margin-top--large"
                   type="button"
-                  disabled={isDisabled}
+                  disabled={isDisabled || this.state.otp.trim() === ""}
```

Comment:
@Binu42 thanks! `otp` has already been destructured on line 61: https://github.com/devfolioco/react-otp-input/pull/149/files#diff-0fb9e0715be10735ccba9436592496eeR61 so no need to have this.state here.

Reply:
@apollonian thanks for the review, I've updated it.